ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -53,7 +53,7 @@
           - ${{sha1}}
           builders:
             - builder-e2e
-          trigger_phrase: ^(?!Thanks for your PR).*/test-(e2e|all).*
+          trigger_phrase: ^(?!Thanks for your PR).*/test-(e2e|all$).*
```

Comment:
I suppose we should remove `.*`? and change it as /test-(e2e|all)$ to ensure only test-e2e and test-all to match the trigger? please check other triggers as well.

Reply:
@luolanzone With this change, only "/test-all" requires an exact match. For other job-specific triggers (e.g., e2e or conformance), users often leave combined comments like "/test-e2e/test-conformance", I feel those should continue to trigger their respective jobs. The issue we’re fixing is "/test-all-feature-conformance" was incorrectly triggering the "test-all" job.

Reply:
maybe what @XinShuYang means is:
```
/test-e2e
/test-conformance
```
which is quite common to have in a single comment (in order to trigger both jobs with one comment)